src/config.cpp

```diff
@@ -2273,17 +2273,43 @@ static int updateMaxclients(long long val, long long prev, const char **err) {
             }
             return 0;
         }
+	/* Change the SetSize for the current thread first. 
+	 * If any error, return the error message to the client, otherwise, continue to do the same for other threads */
+        if ((unsigned int) aeGetSetSize(aeGetCurrentEventLoop()) <
```

Comment:
Just a few formatting issues and we'll be good to go:

1) Your mixing tabs and spaces resulting in inconsistent indentation
2) Use this style for new code:
if (x > y) {
   ...
}
3) Don't try to keep lines 80 chars wide, modern monitors are much wider than tall.  Use judgement when to spill to a new line, but in general the old 80 char standard is far too narrow.

Reply:
Thanks John for the feedback. 
I corrected the format issues as you suggested.